anomalib/core/model/inference.py

```diff
@@ -292,15 +292,13 @@ def post_process(self, predictions: np.ndarray, meta_data: Optional[Dict] = None
             anomaly_map = np.log(anomaly_map)
             anomaly_map = (anomaly_map - meta_data["pixel_mean"]) / meta_data["pixel_std"]
             anomaly_map -= (meta_data["image_mean"] - meta_data["pixel_mean"]) / meta_data["pixel_std"]
-            if "threshold" in meta_data.keys():
```

Comment:
any reason why you decided to remove the if statement?

Reply:
If the `threshold` key is missing from the meta data, I'd rather have the code fail with an error than return the anomaly map without finishing the conversion to normalized scores (I learned the hard way).